test/args_test.dart

```diff
@@ -112,15 +113,20 @@ void main() {
     test(
         'throws ArgumentError if splitCommas is passed with allowMultiple: '
         'false', () {
-      var parser = new ArgParser();
-      throwsIllegalArg(() => parser.addOption('flummox', splitCommas: true));
-      throwsIllegalArg(() => parser.addOption('flummox', splitCommas: false));
+      var parser = ArgParser();
+      throwsIllegalArg(() => parser.addOption(
```

Comment:
Now that there's a line comment in there, I think this is clearer as a `{ ... }` lambda. Below too.

Reply:
ack